ShareX/ShareX.csproj

```diff
@@ -1,17 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
+    <TargetFramework>net80-windows10.0.26100.0</TargetFramework>
```

Comment:
Not sure if its intentional but in other csproj files you had `net80-windows`

Reply:
Hah, yeah.. I never remember to write explanations in my commits 🤦‍♂️

I didn't know about this before, but this is the 'replacement' of the reference to `Microsoft.Windows.SDK.Contracts`. It specifies what version of the Windows APIs are available (used [here](https://github.com/ShareX/ShareX/blob/ec3ce9caebb4ac5a44d7e95e76f1e8f95cae049f/ShareX/Tools/OCR/OCRHelper.cs#L35-L38) for example). [Here's](https://learn.microsoft.com/en-us/dotnet/standard/frameworks#os-version-in-tfms) a section about this in the .NET docs.